include/swift/Basic/Features.def

```diff
@@ -397,6 +397,9 @@ EXPERIMENTAL_FEATURE(ReinitializeConsumeInMultiBlockDefer, false)
 
 EXPERIMENTAL_FEATURE(SE427NoInferenceOnExtension, false)
 
+// Enable trailing comma for comma-separated lists.
+EXPERIMENTAL_FEATURE(TrailingComma, true)
```

Comment:
Please switch the argument here to `false`; this controls whether the experimental feature is enable-able in production compiler builds, which this experimental feature shouldn't be. You'll need to `// REQUIRE: asserts` in any tests that use the experimental flag, which is fine!